packages/kit/src/packaging/index.js

```diff
@@ -1,8 +1,10 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import micromatch from 'micromatch';
 import { createRequire } from 'module';
+
+import micromatch from 'micromatch';
 import { preprocess } from 'svelte/compiler';
+
```

Comment:
And here

Reply:
This one, I think I forgot to condense them from the previous commits, but might as well 😄